sqlc-gen-ftl/src/main.rs

```diff
@@ -0,0 +1,24 @@
+mod protos;
+mod plugin;
+
+use std::io::{Read, Write};
+pub use plugin::Plugin;
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let mut input_bytes = Vec::new();
+    if std::io::stdin().read_to_end(&mut input_bytes).is_err() {
```

Comment:
Is this idiomatic for the plugins? Typically you would also display the error that caused the failure.